client/components/contributepage/InfoSection.js

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import styled from "styled-components";
+
+const Article = styled.article`
+  display: grid;
+  place-items: center center;
+  line-height: 24px;
+  a {
+    color: blue;
+  }
+`;
+
+const BigList = styled.ol`
+  max-width: 1000px;
+  background: ${props => props.theme.colors.light};
+  color: ${props => props.theme.colors.grey};
+  margin: 64px 0;
+`;
+
+const BigItem = styled.li`
+  padding: 16px 0;
+  list-style-type: none;
+  list-style-position: inside;
+`;
+
+const ItemTitle = styled.h3`
+  display: inline-block;
+  font-size: 31px;
+  font-weight: bold;
+  margin: 16px 0;
+`;
+
+const ItemContent = styled.div`
+  font-size: 18px;
+  margin: 18px 32px;
+  font-weight: normal;
+  ol li {
+    list-style-type: decimal;
+    list-style-position: inside;
+  }
+`;
+
+const CodeBlock = styled.code`
+  white-space: pre-wrap;
+  display: inline-block;
+  background: ${props => props.theme.colors.lightGrey};
+  background: #ebeff3; /* props.theme.colors.lightGrey is currently #ccc - outside of design spec */
```

Comment:
@cMikolai Colors should preferably come from themeprovider.
Do we add a new color for this one, or do we replace one of the existing greys (#ccc or #333) ?